lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -2571,9 +2547,16 @@ public function lock($entity, int $lockMode, $lockVersion = null): void
     /**
      * Gets the CommitOrderCalculator used by the UnitOfWork to order commits.
      *
+     * @deprecated use createCommitOrderCalculator() instead
```

Comment:
What's the reason for this deprecation? We usually don't introduce deprecations in bugfix releases and this one does not seem to be necessary.

Reply:
Reverted